packages/slate-tables/src/TablesEditor.ts

```diff
@@ -50,7 +50,7 @@ export namespace TablesEditor {
         return TableCommands.insertRow(editor, location, 'bellow');
     }
 
-    export function isTablesEditor(editor: Editor): editor is TablesEditor {
-        return 'isTableCellNode' in editor;
+    export function isTablesEditor(editor: unknown): editor is TablesEditor {
+        return typeof editor === 'object' && editor !== null && 'isTableCellNode' in editor;
```

Comment:
I think it was because of [this line](https://github.com/prezly/slate/blob/5effad9b2dd794fdbb3747dfceb44952f3a61ff2/packages/slate-tables/src/queries/isHeaderCell.ts#L31) as it otherwise doesn't allow passing both `TablesEditor` and `TableNode` types.

Any ideas how to approach this without using the `unknown`?

Reply:
How about this? 

```ts
import { isEditor } from '@udecode/plate-common';

isEditor(editorOrTable) && TablesEditor.isTablesEditor(editorOrTable)
```